Ukraine Cities

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+class Program
```

Comment:
Push not a single file, but create a folder for this task (CitySearch). Then copy or create new C# project here with .csproj file etc